apps/docs/app/ui/layout/header/Header.tsx

```diff
@@ -96,6 +99,12 @@ const Header = () => {
     const [isMobileMenuOpen, setIsMobileMenuOpen] = useState(false);
     const isMobile = useIsMobile("48rem");
 
+    useEffect(() => {
+        if (logrocketAppId) {
+            LogRocket.init(logrocketAppId);
+        }
+    }, []);
```

Comment:
I know you didn't do it, but I'm wondering if the one below should have used [] as well for a dependency so that it's only called on load. @fraincs , what do you say?
Then we would only need one useEffect for both.

Reply:
This would need to be validated but I under the assumption that since the menu can open / close anytime not only on page load we would not want that?

Reply:
right, I looked at it again and isMobileMenuOpen probably should have been in the dependency array

Reply:
soo if you want to do that Victor and test real quick, that would be nice :) but if Francis would rather do this at some point, then I will approve.

Reply:
Added in the array and tested locally